src/common/config.ts

```diff
@@ -264,6 +271,29 @@ function parseCliConfig(args: string[]): CliOptions {
     return parsed;
 }
 
+export function warnAboutDeprecatedCliArgs(
+    args: CliOptions &
+        UserConfig & {
+            _?: string[];
+        },
+    warn: (msg: string) => void
+): void {
+    console.log(args);
+    let wasWarned = false;
+    // the first position argument should be used
+    // instead of --connectionString, as it's how the mongosh works.
+    if (args.connectionString) {
+        wasWarned = true;
+        warn(
+            "The --connectionString argument is deprecated. Prefer using the first positional argument for the connection string or the MDB_MCP_CONNECTION_STRING environment variable."
+        );
+    }
+
+    if (wasWarned) {
+        warn("Refer to https://www.mongodb.com/docs/mcp-server/get-started/ for setting up the MCP Server.");
```

Comment:
```suggestion
        warn("Refer to https://www.mongodb.com/docs/mcp-server/get-started/ for setting up the MCP Server.");
```
is there a difference here?

Reply:
> if we add more deprecated arguments
might make sense to do this when we actually get to that otherwise we're optimizing on an assumption.
alternatively if you feel strongly about it, could rename`wasWarned` to `usedDeprecatedArgument` so that flow is a bit clearer